Client/src/view/BattleView/HandBox.java

```diff
@@ -108,17 +114,21 @@ private void updateNext() {
             cardAnimation = null;
         }
 
+         */
+
+        CardAnimation cardAnimation = null;
+
         imageView2.setImage(nextRingSmoke);
 
-        if (cardAnimation != null) {
+        if (cardAnimation != null ) {
             next.setOnMouseEntered(mouseEvent -> {
                 if (battleScene.isMyTurn()) {
-                    cardAnimation.inActive();
+                    //cardAnimation.inActive();
                     imageView2.setImage(nextRingShine);
                 }
             });
             next.setOnMouseExited(mouseEvent -> {
-                cardAnimation.pause();
+                //cardAnimation.pause();
                 imageView2.setImage(nextRingSmoke);
             });
         }
```

Comment:
This code can be deleted since `cardAnimation` will always be `null` (and we don't need `cardAnimation` anymore either).